openshift/.pipeline/Jenkinsfile

```diff
@@ -85,6 +86,19 @@ pipeline {
          -p NAMESPACE=${TOOLS_NAMESPACE} | oc apply -f - -n ${TOOLS_NAMESPACE}"
       StartBuildAndWait("${SAMPLE_ADAPTER}-build", TOOLS_NAMESPACE)
       StartBuildAndWait("${SAMPLE_ADAPTER}-runtime", TOOLS_NAMESPACE)
+     },
+     'ICBCAdapter': {
```

Comment:
Can we name this one IcbcRestAdapter:

![image](https://user-images.githubusercontent.com/51387119/70480305-3c74f000-1a94-11ea-95b5-2a97aaf69d04.png)

Reply:
I have renamed the artifacts to reflect the title 'ICBC Rest Adapter'

Reply:
Thanks @pdittaro